### DISCUSSION TITLE
Where is the `tokyo-night` theme?

### DISCUSSION THREAD

**Discussion by plutonium-239** [tconbeer/harlequin#717] [Q&A, Troubleshooting, Support] [answered]
The website shows a [screenshot of `harlequin` in the tokyo-night theme](https://harlequin.sh/_app/immutable/assets/tokyo-night.61osJNHC.svg) on the front page.

However, I get an error when I try to specify that in the program. 
Moreover, it is not a [`pygments` style](https://pygments.org/styles/), and searching for it doesn't show any results in this repo.

For reference, here is the error:
![image](https://github.com/user-attachments/assets/70353524-f703-4abd-87ca-77e2156b64a4)

**Comment by tconbeer** [comment]
We're in the middle of migrating themes. This is my bad, but the website shows the docs for the upcoming 2.0 release of Harlequin, which will use a different set of themes. Harlequin 1.X still uses Pygments styles as themes.

**Reply by tconbeer** [reply, marked as answer]
@plutonium-239 v2.0.0 has been released. Please update to the latest version to use the tokyo-night theme.
